recipes/fABBA/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "fABBA" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf0672c2d0be449627380f51a7e695ddd43bf548a292de9dcfe8562e256ff95a
+
+
+build:
+  noarch: python
```

Comment:
```suggestion
```
This is not a noarch package because it contains a compiled module using cython.

Reply:
Fix, thanks for pointing out this, I copy the recipe from our another pure python package and forget to change it.